lms/static/coffee/src/instructor_dashboard/student_admin.coffee

```diff
@@ -223,6 +224,31 @@ class @StudentAdmin
           full_error_message = interpolate_text(error_message, {student_id: unique_student_identifier})
           @$request_response_error_ee.text full_error_message
 
+  # Mark a student to skip entrance exam
+    @$btn_skip_entrance_exam.click =>
+      unique_student_identifier = @$field_entrance_exam_student_select_grade.val()
+      if not unique_student_identifier
+        return @$request_response_error_ee.text gettext("Please enter a student email address or username.")
+      confirm_message = gettext("Mark student '{student_id}' to skip entrance exam?")
+      full_confirm_message = interpolate_text(confirm_message, {student_id: unique_student_identifier})
+      if window.confirm full_confirm_message
+        send_data =
+          unique_student_identifier: unique_student_identifier
+
+        $.ajax
+          dataType: 'json'
+          url: @$btn_skip_entrance_exam.data 'endpoint'
+          data: send_data
+          type: 'POST'
+          success: @clear_errors_then ->
+            success_message = gettext("Student '{student_id}' is marked to skip entrance exam.")
+            full_success_message = interpolate_text(success_message, {student_id: unique_student_identifier})
+            alert full_success_message
+          error: std_ajax_err =>
+            error_message = gettext("Error marking student '{student_id}' to skip entrance exam. Make sure student identifier is correct.")
```

Comment:
Suggested revisions:

We couldn't find this student ('{student_id}'). Make sure that the student's username or email address is correct and try again.

OR

An error occurred. Make sure that the student's username or email address is correct and try again.